src/main.ts

```diff
@@ -35,6 +36,12 @@ async function run(): Promise<void> {
         await bump.Deploy.run(cliParams.concat(docCliParams));
         break;
       case 'diff':
+        const baseFile = await new Repo().getBaseFile(file);
+
+        if (baseFile) {
+          cliParams.unshift(baseFile);
```

Comment:
Just to be sure, is this part related to https://github.com/bump-sh/cli/pull/141#pullrequestreview-770457564 ?

Or to ask better question, would the link commented have impact here ?

Reply:
Yes exactly, here I have to use `unshift` (instead of `push) on the array to put the `baseFile` **first** in the CLI params to obtain the correct diff. If we add the `--previous` flag, I wouldn't need to worry about the params order.